pom.xml

```diff
@@ -4,7 +4,8 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.23</version>
+        <version>2.31-20170612.235547-1</version> <!-- TODO https://github.com/jenkinsci/plugin-pom/pull/67 -->
```

Comment:
If you don't need this right away, do you want to wait until the next version is cut?  This plugin isn't released frequently, so it might be worth waiting until the plugin-pom is updated.